src/app/client/src/components/NetworkTableCard.tsx

```diff
@@ -8,73 +9,121 @@ import {
   TableHeader,
   TableRow,
 } from "@/components/ui/table";
-import {
-  Card,
-  CardHeader,
-  CardTitle,
-  CardContent,
-  CardDescription,
-} from "@/components/ui/card";
+import { Card, CardHeader, CardTitle, CardContent } from "@/components/ui/card";
+import { Switch } from "@/components/ui/switch";
+import { Label } from "@/components/ui/label";
 import type { FeederMessage } from "../../../types/client-types";
+import PcapMetaDataModal from "./PcapMetaDataModal";
+
+const classMap = {
```

Comment:
there is another model with less classes that has a different class map.  we need a good way to track which model is on the feeder for display on the gui